### ISSUE TITLE
About the cyclist on KITTI test

### ISSUE THREAD

**Issue by pummi823** [AutoVision-cloud/SA-Det3D#8]
Dear author, i train your pvrcnn_fas.yaml，only cyclist，and split the training set to 80% and 20% for training and validating, then choose the best ckpt to run the test set, but only get: 75.59 % | 58.31 % | 52.01 % for easy, mod, hard , which are far away from your results in your paper. Can you tell me how to reproduce your results on cyclist?? Thank you!
